### PR TITLE
FE - Navtop, useAuth.js, useAxios.js

### PR DESCRIPTION
**Pull request by Mpanasetckiy**

### CODE REVIEW

File: frontend_/src/components/Menu/MenuView.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import config from "../../config.json";
+
+const MenuView = () => {
+    const { routes } = config;
+
+    const items = [
+        {
+            key: 1,
+            label: <Link to={routes.userProfilePage.path}>User Name</Link>,
```
Comment (serhii-fsi) [inline frontend_/src/components/Menu/MenuView.jsx:22]: we should use getRoute(routes.userProfilePage.path, some params)

File: frontend_/src/components/NavTop/NavTop.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import NavTopView from "./NavTopView";
+
+/**
+ *
+ * @param {object} props
+ * @param {string} props.title
+ * @param {string} props.logo
```
Comment (serhii-fsi) [inline frontend_/src/components/NavTop/NavTop.jsx:7]: Also we need some logic to change logo for "go back" Link or button which force Router to navigate back.

We can and should hardcode this logo icon inside NavView

File: frontend_/src/components/NavTop/NavTopView.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+header {
```
Comment (serhii-fsi) [inline frontend_/src/components/NavTop/NavTopView.css:1]: NavTopView-header

File: frontend_/src/components/NavTop/NavTopView.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import Menu from "../Menu/Menu";
+
+import "./NavTopView.css";
+
+const NavTopView = ({ title, logo }) => {
+    return (
+        <header>
+            <h5>{logo}</h5>
+            <h4>{title}</h4>
```
Comment (serhii-fsi) [inline frontend_/src/components/NavTop/NavTopView.jsx:9]: h1

**Review by serhii-fsi — CHANGES_REQUESTED** [review]

File: frontend_/src/components/NavTop/NavTop.jsx (hunk 2 above)
Comment (Mpanasetckiy) [reply frontend_/src/components/NavTop/NavTop.jsx:7]: I will add it later

File: frontend_/src/components/NavTop/NavTopView.jsx (hunk 4 above)
Comment (Mpanasetckiy) [reply frontend_/src/components/NavTop/NavTopView.jsx:9]: h1 is too big here

**Pull request merged**